docs/concepts/copilot/enable-copilot.md

```diff
@@ -12,32 +12,40 @@ tags:
 
 From the kubernetes cluster where Prophecy services are running:
 
-1. The `prophecy-operator` service must be running version `3.1.0.0` or greater.
-2. The `athena` Prophecy service should be running version `3.1.0.0` or greater. This is only required if using the Pipeline approach to the upgrade.
-3. In the `ProphecyCluster` CR, do the following:
+1. The `prophecy-operator`, `edweb`, `metagraph`, and other Prophecy services must be running version `3.1.0.0` or greater.
+2. The above services should all be running the same release-version as the `copilot` service defined in the image below, eg `3.1.0.0`
+3. The `athena` Prophecy service should be running version `3.1.0.0` or greater. This is only required if using the Pipeline approach to the upgrade.
+4. In the `ProphecyCluster` CR, do the following:
 
 - add `copilot` to `enabledapps`
 - add `image` section if required as below, else defaults to `latest`
```

Comment:
We can remove this and always suggest to provide a version. @prashanth26 if nothing is specified it will point to our internal ECR right?